cvat/apps/engine/serializers.py

```diff
@@ -595,6 +595,9 @@ class RqStatusSerializer(serializers.Serializer):
     message = serializers.CharField(allow_blank=True, default="")
     progress = serializers.FloatField(max_value=100, default=0)
 
+class RqIdSerializer(serializers.Serializer):
+    rq_id = serializers.CharField()
```

Comment:
The parameter should not be called `rq_`, because it can uncover implementation details to a potential attacker. I suggest to change it to `upload_id` or `queue_id` instead. It should also be limited to some size like 40 symbols. It will be even better if the value is a hashed string.

Reply:
> The parameter should not be called rq_, because it can uncover implementation details to a potential attacker.

The source is open - all details are already out there.

I would prefer if it was called `request_id`, but `rq_id` is already used in other places, and I'd rather avoid gratuitous changes.